packages/ifpack2/test/unit_tests/Ifpack2_UnitTestContainer.cpp

```diff
@@ -476,15 +476,13 @@ TEUCHOS_UNIT_TEST_TEMPLATE_3_DECL(BandedContainer, FullMatrixSameScalar, Scalar,
                          localSuccess, outArg (globalSuccess));
   }
 
-  // FIXME: why is this not working?? It seems that the gather call from X to X_local is failing (maybe due to problems with localRows?)
-  // TODO second call to apply not working due to problems with local rows???
-  //out << "DenseContainer::weightedApply" << endl;
-  //d.putScalar (1.0);
-  //MyContainer->weightedApply (b, y, d);
-  //
-  //out << "Computing results of apply() and weightedApply() "
-  //    << "(they should be the same in this case)" << endl;
-  //TEST_COMPARE_FLOATING_ARRAYS( x.get1dView(), y.get1dView(), 1.0e2*STS::eps () );
+  out << "DenseContainer::weightedApply" << endl;
+  d.putScalar (1.0);
+  MyContainer->weightedApplyMV(b, y, d);
+
+  out << "Computing results of apply() and weightedApply() "
+      << "(they should be the same in this case)" << endl;
+  TEST_COMPARE_FLOATING_ARRAYS( x.get1dView(), y.get1dView(), 1.0e2*STS::eps () );
```

Comment:
Be careful mixing a `double` (1.0e2) with a magnitude type.  That could promote and cause build warnings or even loss of accuracy.  Consider casting 1.0e2 to the magnitude type that `STS::eps()` returns.